src/test/java/model/ByeongTest.java

```diff
@@ -0,0 +1,102 @@
+package model;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.util.HashMap;
+import java.util.Map;
+import model.piece.Byeong;
+import model.piece.Cha;
+import model.piece.Piece;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+public class ByeongTest {
```

Comment:
JUnit5 부터는 테스트 클래스 및 메서드가 public일 필요가 없습니다~
```suggestion
class ByeongTest {
```

Reply:
오 새롭게 알게된 사실이네요 감사합니다~